samples/js/browser/index.html

```diff
@@ -433,7 +433,7 @@ <h2 style="font-weight:500;">Microsoft Cognitive Services</h2>
                     window.console.log(e);
                     statusDiv.innerHTML += "(recognizing) Reason: " + SpeechSDK.ResultReason[e.result.reason] + " Text: " + e.result.text + " Translations:";
 
-                    var language = languageTargetOptions.value.split("-")[0];
+                    var language = languageTargetOptions.value.split("(")[1].substring(0, 2);
```

Comment:
Are we sure that the target language returned is only 2 characters? For some languages, you can specify like zh-hans, zh-hant, or tlh-qaak.